frame/support/test/tests/decl_storage.rs

```diff
@@ -526,47 +526,47 @@ mod test_append_and_len {
 	#[test]
 	fn append_works() {
```

Comment:
kill + append
overwrite + append

would be nice additions to this test

Reply:
> overwrite + append
Is done below in `append_overwrites_invalid_data`

Reply:
Added kill + append